NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -294,9 +294,9 @@ public enum NVActivityIndicatorType: Int {
 
 /// Activity indicator view with nice animations
 public class NVActivityIndicatorView: UIView {
-    private static let DEFAULT_TYPE: NVActivityIndicatorType = .BallSpinFadeLoader
-    private static let DEFAULT_COLOR = UIColor.whiteColor()
-    private static let DEFAULT_PADDING: CGFloat = 0
+    public static var DEFAULT_TYPE: NVActivityIndicatorType = .BallSpinFadeLoader
```

Comment:
I made the defaults public so they can be modified globally. It would be rare that an app would use different types of animation within the same app to remain consistent, so setting the defaults would make being less tedious.